mc-config.sh

```diff
@@ -10,8 +10,9 @@ mc config host add minio http://minio:9000 $MINIO_ACCESS_KEY $MINIO_SECRET_KEY -
 
 mc mb minio/samples/processed/
 mc mb minio/samples/unprocessed/
+mc mb minio/drs/
 
-mc admin config set minio notify_webhook:1 queue_limit="0" endpoint="http://listener:8080/events"
+mc admin config set minio notify_webhook:1 queue_limit="0" endpoint="http://listener:8081/events"
```

Comment:
should be consistent with `LISTENER_PORT` if possible

Reply:
Solved in a49a74f